base/leaky_datastore.go

```diff
@@ -26,7 +27,7 @@ type LeakyDataStore struct {
 var (
 	_ DataStore         = &LeakyDataStore{}
 	_ WrappingDatastore = &LeakyDataStore{}
-	// _ N1QLStore = &LeakyDataStore{} // TODO: Not implemented
+	_ N1QLStore         = &LeakyDataStore{}
```

Comment:
i think adding n1ql support to leakydatastore is not necessary for this PR (and was speculative on main)